openapi/SwarmDebug.yaml

```diff
@@ -92,8 +92,8 @@ paths:
             application/json:
```

Comment:
I think changes to SwarmDebug.yaml can be avoided as we should be incrementing versions here when we plan to not support it going ahead.

Reply:
Added the recomended semver changes to the PR description.